qa/workunits/cephtool/test.sh

```diff
@@ -247,7 +247,7 @@ function test_tiering_agent()
   done
   $evicted # assert
   # the object is proxy read and promoted to the cache
-  rados -p $slow get obj1 /tmp/obj1
+  rados -p $slow get obj1 $TMPFILE
```

Comment:
how about ```rados -p $slow get obj1 - > /dev/null``` since we don't use the result? if we reordered tests at some point ```$TMPFILE``` could already exist